packages/neos-ui/src/manifest.js

```diff
@@ -428,12 +429,31 @@ manifest('main', {}, globalRegistry => {
                 parentElement.appendChild(contentElement);
                 break;
         }
+
+        initializeHoverHandlersInIframe(contentElement, dom.iframeDocument());
+
+        const nodeTypesRegistry = globalRegistry.get('@neos-project/neos-ui-contentrepository');
+        const formattingRulesRegistry = globalRegistry.get('@neos-project/neos-ui-ckeditor-bindings').get('formattingRules');
+
+
+
+        initializeCkEditorForDomNode(contentElement, {
+            byContextPathDynamicAccess: contextPath => selectors.CR.Nodes.byContextPathSelector(contextPath)(store.getState()),
+            calculateEnabledFormattingRulesForNodeType: _calculateEnabledFormattingRulesForNodeType({
+                nodeTypesRegistry,
+                formattingRulesRegistry
+            }),
+            nodeTypesRegistry,
+            formattingRulesRegistry,
+            translations: tmp.translations,
+            persistChange: (...args) => store.dispatch(actions.Changes.persistChange(...args))
```

Comment:
for initializing the CKEditor, you need quite a lot of things.... not sure how to work around here.